src/frontend/src/expr/window_function.rs

```diff
@@ -124,7 +124,7 @@ impl std::fmt::Debug for WindowFunction {
             }
             builder.finish()
         } else {
-            write!(f, "{}() OVER(", self.kind)?;
+            write!(f, "{}({:?}) OVER(", self.kind, self.args)?;
```

Comment:
What about display arguments line in `PlanWindowFunctionDisplay`?

https://github.com/risingwavelabs/risingwave/blob/67ab16bc058397fda7f1c7bee6dfeb99480cde7a/src/frontend/src/optimizer/plan_node/generic/over_window.rs#L60-L75